### ISSUE TITLE
Add copy/cut and paste cells/sheets between USHEET / XLSX within Obisidian

### ISSUE THREAD

**Issue by OMSmolina** [dream-num/obsidian-univer#66]
### Problem

You cannot paste cells or entire sheets into another USHEET or XSLX file within Obsidian.
It also does not allow copying cells or sheets from external files such as Google Spreadsheets, CSV or XSLX

### Solution

Add the ability to copy and paste cells or sheets between files within Obsidian as external files from Google Spreadsheets, XLSX, CSV

**Comment by kyounes** [comment]
Yes, and add the ability to just import a xlsx or libreoffice spreadsheet into univer. Also export from univer to xlsx and/or libreoffice

**Comment by karlsbeard** [comment]
> Yes, and add the ability to just import a xlsx or libreoffice spreadsheet into univer. Also export from univer to xlsx and/or libreoffice

We will provide this function in the next version, just hold a second.
